include/forcing/AorcForcing.hpp

```diff
@@ -14,6 +14,18 @@
 #define CSDMS_STD_NAME_WIND_V_Y "land_surface_wind__y_component_of_velocity"
 #define NGEN_STD_NAME_SPECIFIC_HUMIDITY "atmosphere_air_water~vapor__relative_saturation" // This is not present in standard names, use this for now... may change!
 
+// Recognized Forcing Value Names (in particular for use when configuring BMI input variables)
+// TODO: perhaps create way to configure a mapping of these to something different
+#define AORC_FIELD_NAME_PRECIP_RATE "precip_rate"
```

Comment:
Interestingly... these defines appear to only be used in test code... though the magic strings appear elsewhere (in this file even)... no action necessary here now, but this is probably something that should be refactored, some way, at some point.

Reply:
Yeah, need to keep in mind.